lib/sentry_logging.rb

```diff
@@ -17,8 +17,11 @@ def log_message_to_sentry(message, level, extra_context = {}, tags_context = {})
   # rubocop:disable Metrics/CyclomaticComplexity
   # rubocop:disable Metrics/PerceivedComplexity
   def log_exception_to_sentry(exception, extra_context = {}, tags_context = {}, level = 'error')
+    level = exception.sentry_type if exception.is_a?(Common::Exceptions::BaseError)
+    # Why is this line below required?!?!? it will override whats specified in exception.en.yml; Why info??
     level = 'info' if extra_context.is_a?(Hash) && client_error?(extra_context[:va_exception_errors])
     level = normalize_level(level)
+    return if level == 'none'
```

Comment:
this is also going to prevent logging to Rails.logger -- I don't know if we want to do that. :/

Reply:
I think we want to continue to log to Rails.logger. Eventually, _everything_ sent to Rails.logger will get logged to Loki. That isn't today's reality, but it is in our Q4 2020 plans. It's possible I don't fully understand the context though, so I'm leaving room for me to be wrong. 😸

Reply:
I can tweak it so that it just skips Sentry but still does rails logging, but if log level is "none" what should the Rails logger "level" be for semantic logger or Loki?

Reply:
alternately, what I could do instead is to have two or three different log_levels, one for each of: Rails, Loki, Sentry.

Reply:
I don't think that's what we're today for the case of :none as evidenced by specs still passing. I could be wrong.